### ISSUE TITLE
Installation fails on VB17

### ISSUE THREAD

**Issue by davidress** [CMRR-C2P/MB#101]
Hi,

I tried to perform the installation on a 32-channel upgraded TIM Trio. This machine already had the previous version of this multi-band sequence installed, but for this version, the last three steps of the installation batch file failed. Each failure stated something about a ".i87" file, which is a file type that I'm not familiar with. We rebooted the scanner and  tried again; we got the same result. We shut the scanner down, and then tried again; same result. Can you provide any guidance, please?

**Comment by eauerbach** [comment]
I'm not sure what you mean by an upgraded Trio. Has it been upgraded to VB19A or is it still on VB17A? The .i87 binaries are new for VB19A and VB20P. In any case, are you sure you have the latest zip for your software version from the web site (http://www.cmrr.umn.edu/multiband/)? If you downloaded a copy prior to June 1, please download a new copy since I made a small fix to the installation batch file which fixes some problems if you try to install from a directory with a space anywhere in the pathname.

**Comment by davidress** [comment]
By "upgraded", I mean that it used to be an 8-channel TIM Trio that was upgraded to 32-channel capability. This machine runs VB17A.

I installed the very latest B17 download: CMRR_MB_EPI_VB17A_R012b. I looked at the .bat file, and, sure enough, it contains commands to install a set of .i87 files at the start. This is the only place that the .i87 files are mentioned, so I am guessing that this is a bug. I will try to re-run the installation with those lines removed. Will be back with the result.

Cheers,
DR


David Ress
davidress@gmail.com
